### ISSUE TITLE
Transition from edx-platform project to openedx-translations project

### ISSUE THREAD

**Issue by ehuthmacher** [openedx/wg-translations#23]

**Comment by ehuthmacher** [comment]
@OmarIthawi Has the edx-platform project been converted to "read-only"?

**Comment by ehuthmacher** [comment]
@OmarIthawi We currently only have 18 languages in the openedx-translations project. However, we will need to shift all 133 languages that were part of the edx-platform project to the openedx-translations project so that the community can still work on their desired languages. This must be done via API: https://developers.transifex.com/reference/patch_projects-project-id-relationships-languages

**Comment by OmarIthawi** [comment]
> @OmarIthawi Has the edx-platform project been converted to "read-only"?

Yes, @shadinaif has confirmed that all resources are now read-only:

![image](https://github.com/openedx/wg-translations/assets/645156/38c21b79-d276-44ed-87fd-c350f0b20335)

> @OmarIthawi We currently only have 18 languages in the openedx-translations project. However, we will need to shift all 133 languages that were part of the edx-platform project to the openedx-translations project so that the community can still work on their desired languages. This must be done via API: https://developers.transifex.com/reference/patch_projects-project-id-relationships-languages

Sorry for the delay, we'll take care of that ASAP this week.

**Comment by ehuthmacher** [comment]
Proposal: sync languages as the requests come in

**Comment by ehuthmacher** [comment]
We need to develop a process for accepting new languages: current process - TX requests get approved upon submission - proposal: request for min translation commitment - confirm commitment via agreement @ehuthmacher + be sure to allow several weeks to fix invalid translations to activate the syncing

**Comment by ehuthmacher** [comment]
@OmarIthawi will propose a policy for requirements to add new languages to openedx-translation project. @brian-smith-tcril please review upon receipt.

**Comment by OmarIthawi** [comment]
I'm adding the policy in the pull request below:

 - https://github.com/openedx/docs.openedx.org/pull/469

**Comment by ehuthmacher** [comment]
We need to update all correlating documentation.

**Comment by ehuthmacher** [comment]
@ehuthmacher Add links to dated documentation

**Comment by ehuthmacher** [comment]
Reviews have enhanced. However, people are still having trouble accessing the new project - double verification needed.

**Comment by ehuthmacher** [comment]
Ticket open with TX to identify ongoing issues

**Comment by ehuthmacher** [comment]
@OmarIthawi Please see inquiry below from Tashi Dhondup - are people still accessing the edx-platform project and are the syncing errors still remaining:

I would like to ask a couple of questions about the edX platform. Specifically, I have noticed that there are two other entities named edX-platform and edX-platform.js in openedx-translations.

I have already translated the edX-platform into Tibetan with more than 2000 strings. (This version of edX-platform is not in openedx-translations and may be an older version.)
There are around 7000 strings in both edX-platform and edX-platform.js in openedx-translations. (These two are in openedx-translations and might be newer versions.)
My main question is, what is the difference between the edX-platform I translated and the two entities named edX-platform and edX-platform.js in openedx-translations? Is there a way to merge or transfer my Tibetan translations from the edX-platform I worked on to the edX-platform and edX-platform.js in openedx-translations? If this is possible, I would be very grateful for your assistance in making it happen.

Additionally, I have a question about using AI in Transifex. How can I use the AI feature in Transifex? It says, "Please contact your administrator to add them to your plan." If possible, could you provide some guidance on this matter? I have attached two illustrations below to make the issue clear.
Thank you very much for your patience and kindness.

Best regards,
Dhondup

**Comment by ehuthmacher** [comment]
We may need to archive the edx-platform project - people are still accessing it and are not sure where to conduct their efforts. Hence, the performance rates of the openedx-translations project are low.

**Comment by OmarIthawi** [comment]
@ehuthmacher I support archiving the project.

Nevertheless, my understanding that Dhondup is referring to `resources` when he mentions edx-platform and edx-platform.js.

I'll reply to his email asking for clarification.

**Comment by ehuthmacher** [comment]
Proceed with archiving the project.

**Comment by ehuthmacher** [comment]
@ehuthmacher Will check if maintainers can be automatically added from the Master project
